src/app/shared/label-dropdown/label-dropdown.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { LabelService } from '../../core/services/label.service';
+import {FormGroup, AbstractControl} from '@angular/forms';
+import { Label } from '../../core/models/label.model';
+
+@Component({
+  selector: 'app-label-dropdown',
+  templateUrl: './label-dropdown.component.html',
+  styleUrls: ['./label-dropdown.component.css']
+})
+export class LabelDropdownComponent implements OnInit {
+  @Input() dropdownControl: AbstractControl;
```

Comment:
The `attributeName` and `dropdownControl` seem to have the same values repeated throughout. Perhaps a way we could enhanced this is removing the `dropdownControl` as an `@Input` and simply running the following in `ngOnInit()`

```typescript
ngOnInit() {
  // ...
  this.dropdownControl = this.dropdownForm.get(this.attributeName);
}
```

My preliminary tests on this seem to be working, maybe this way we can reduce the code verbosity a little?